augur/export_v2.py

```diff
@@ -861,7 +861,9 @@ def register_parser(parent_subparsers):
         title="OPTIONAL SETTINGS"
     )
     optional_settings.add_argument('--minify-json', action="store_true", help="export JSONs without indentation or line returns")
-    optional_settings.add_argument('--include-root-sequence', action="store_true", help="Export an additional JSON containing the root sequence (reference sequence for vcf) used to identify mutations. The filename will follow the pattern of <OUTPUT>_root-sequence.json for a main auspice JSON of <OUTPUT>.json")
+    root_sequence = optional_settings.add_mutually_exclusive_group()
+    root_sequence.add_argument('--include-root-sequence', action="store_true", help="Export an additional JSON containing the root sequence (reference sequence for vcf) used to identify mutations. The filename will follow the pattern of <OUTPUT>_root-sequence.json for a main auspice JSON of <OUTPUT>.json")
+    root_sequence.add_argument('--include-root-sequence-inline', action="store_true", help="Export the root sequence (reference sequence for vcf) used to identify mutations as part of the main dataset JSON. This should only be used for small genomes for file size reasons.")
```

Comment:
_non-blocking_

It would be great to have Cram tests for these flags.

Reply:
Yeah - I may get around to this, but (as you probably know) there are no existing tests for root-sequence export

Reply:
Sorry I'm late to this, just trying to take advantage of this feature and looked into the PR as a result!

I've got a few questions!
- Why are we worried about file size? 
- Should the same "should only be used for small genomes" apply to the sidecar file as well?
- Might be good to specify what "small genome" means, how many kb/mb roughly?

Reply:
> Why are we worried about file size?

Increases in the size of the main JSON file directly increase the initial dataset load time in Auspice. This is, I've always understood it, the main reason for the sidecars (vs. inlining everything): sidecars will be loaded in the background.

> Should the same "should only be used for small genomes" apply to the sidecar file as well?

Not nearly as much since it can be loaded in the background.

> Might be good to specify what "small genome" means, how many kb/mb roughly?

Sure would!